styles/styles.css

```diff
@@ -12,10 +12,10 @@
  * governing permissions and limitations under the License.
  */
 
-:root {
+:root, .elsie-base-design.elsie-base-design {
```

Comment:
Would like to fix this specificity issue. Each DropIn injects a copy of the base tokens, but they are lazy-loaded so technically has higher specificity than the styles.css file.